app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+
+from app.errors import NotVaccinatedError
+
+from app.errors import OutdatedVaccineError
+
+from app.errors import NotWearingMaskError

```

Comment:
The import is not formatted correctly, in case you import multiple entities from the same module, there is no need to duplicate the module several times
```suggestion
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError,
)
```